src/main/scala/kinesis/mock/KinesisMockService.scala

```diff
@@ -24,6 +25,15 @@ object KinesisMockService extends IOApp {
           "Logging Cache Config"
         )
         cache <- Cache(cacheConfig)
+        _ <- cacheConfig.initializeStreams.toList.flatten.map(s =>
+          for {
+            _ <- logger.info(
+              s"Initializing stream '${s.streamName}' " +
+                s"(shardCount=${s.shardCount})")
+            _ <- cache.createStream(s, context, false)
+            _ <- IO.sleep(cacheConfig.createStreamDuration)
+          } yield {}
+        ).parSequence_
```

Comment:
If we're going to do a parallel strategy, we should batch these up in groups of 5. There's a limit on having 5 of these requests running at once:

https://github.com/etspaceman/kinesis-mock/blob/main/src/main/scala/kinesis/mock/cache/CacheSemaphores.scala#L29

Also I think you can just do a `parTraverse_` instead of `.map(...).parSequence_`